packages/opencensus-exporter-prometheus/src/prometheus-stats.ts

```diff
@@ -0,0 +1,225 @@
+/**
+ * Copyright 2018, OpenCensus Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {AggregationType, DistributionData, ExporterConfig, logger, Logger, Measurement, MeasureUnit, StatsEventListener, Tags, View} from '@opencensus/core';
+import * as express from 'express';
+import * as http from 'http';
+import {Counter, Gauge, Histogram, Metric, Registry} from 'prom-client';
+
+export interface PrometheusExporterOptions extends ExporterConfig {
+  /** App prefix for metrics, if needed - default opencensus */
+  prefix?: string;
+  /**
+   * Port number to Prometheus server
+   * Default registered port is 9464:
+   * https://github.com/prometheus/prometheus/wiki/Default-port-allocations
+   */
+  port?: number;
+  /** Define if the Prometheus server will be started - default false */
+  startServer?: boolean;
+}
+
+/** Format and sends Stats to Prometheus */
+export class PrometheusStatsExporter implements StatsEventListener {
+  static readonly DEFAULT_OPTIONS = {
+    port: 9464,
+    startServer: false,
+    contentType: 'text/plain; text/plain; version=0.0.4; charset=utf-8',
+    prefix: 'opencensus'
+  };
+
+  private logger: Logger;
+  private prefix: string;
+  private port: number;
+  private app = express();
+  private server: http.Server;
+  // Registry instance from Prometheus to keep the metrics
+  private registry = new Registry();
+
+  constructor(options: PrometheusExporterOptions) {
+    this.logger = options.logger || logger.logger();
+    this.port = options.port || PrometheusStatsExporter.DEFAULT_OPTIONS.port;
+    this.prefix =
+        options.prefix || PrometheusStatsExporter.DEFAULT_OPTIONS.prefix;
+
+    /** Start the server if the startServer option is true */
+    if (options.startServer) {
+      this.startServer();
+    }
+  }
+
+  /**
+   * Not used because to register metrics it's necessary some information only
```

Comment:
Nit: `to register metrics it's necessary some information` -> `registering metrics requires information that is`

Reply:
Ok